src/screens/BeersFavourite/FavouriteBeers.jsx

```diff
@@ -31,20 +32,26 @@ class FavouriteBeersComponent extends PureComponent {
   };
 
   deleteBeer = (id) => {
+    const clicked = true;
+    this.setState({ clicked });
```

Comment:
No need to create new variable. I'd create new method for that:
```
toggleDeleteButtonClicked = () => {
  this.setState(prevState => ({
    deleteButtonClicked: !prevState.deleteButtonClicked
   }));
 };
```

Reply:
Done